src/components/breadcrumbs.vue

```diff
@@ -44,11 +46,13 @@ defineProps({
 }
 
 .breadcrumb-item {
+  @include text-overflow-ellipsis;
   font-size: 14px;
-  color: #bbb;
+  max-width: 275px;
 
-  [class*='icon'] {
+  a [class^="icon-"] {
```

Comment:
I think it's preferable to avoid `!important`, so I went ahead and made the selector more specific.